frontend/modules/browse/components/Filter.js

```diff
@@ -32,23 +32,29 @@ export default injectIntl(React.createClass({
       }
     });
 
-    var _onClick = this._onClick;
-    var active = this.props.active;
-    var items = this.props.data.map(function(item) {
-      var classNames = "list-group-item";
-      if (item.title == active) {
-        classNames += " active";
+    const active = this.props.active;
+    const items = this.props.data.map((item) => {
+      let classNames = ["list-group-item"];
+      if (item.slug == active) {
+        classNames += [" active"];
       }
+
+      if (item.total == 0) {
+        return null;
+      }
```

Comment:
Should this check be moved to the backend? IE, only send back data that has more than 1 result and let the frontend just display everything it gets back? Seems like the backend is returning data that is never used.